bridges/zap/version.go

```diff
@@ -0,0 +1,6 @@
+package zap // import "go.opentelemetry.io/contrib/bridges/zap"
```

Comment:
All code and documentation files need to have a license header.

```suggestion
// Copyright The OpenTelemetry Authors
// SPDX-License-Identifier: Apache-2.0

package zap // import "go.opentelemetry.io/contrib/bridges/zap"
```